docs/view_components/flow_chart.rst

```diff
@@ -1,11 +1,40 @@
-Component title
----------------
+.. _flowchart-label:
+
+Flow-chart
+----------
+The flow chart is a view component that shows all the sequential steps of the tool in order to get results.
+
+A flow chart view-component has a number of advantages:
+
+* Helps in sequentially progressing through the modeling and simulation process
+* Allows the user to jump between different steps of the simulation
+* Brings a design consistency and uniformity to the UI of the tool
+* Splits the tool in different views which increases the user friendliness 
 
 Attributes
 ^^^^^^^^^^
 
+* Clickable buttons that are identical throughout the different views
+* Ribbon shaped, clickable icons/buttons placed sequentially as per the simulation progression order.
+* Each button is a step in the simulation process
+* Each step has to be completed one at a time. Once a step has met its minimum requirements, the button of the next step becomes clickable. Therefore, the user can come back to the steps he has already seen but he cannot click a random step. 
+
```

Comment:
These are not attributes: attributes for flow chart are
- ordered list of steps
- index of current step

The fact that the steps can be clicked on is defined within "Actions", the fact that the steps are button or links is described under Rendering of the view-component